templates/monorepo/README.md

```diff
@@ -0,0 +1,85 @@
+# TSDX Monorepo User Guide
+
+## Usage
+
+This monorepo is setup for a dummy `@mono/` NPM organization. There are 2 packages by default:
+
+- `@mono/react` - A placholder React component
+- `@mono/utils` - A utils packages
+
+Unlike other TSDX templates, the developer experience for this template is currently a bit more manual.
+
+Your first order of business will be to search and replace `@mono` for the npm organization of your own.
```

Comment:
Also need to change author/username and LICENSE file

Reply:
Lerna syncs the license from root prior to publishing for each package. https://github.com/lerna/lerna/commit/58635642ae929d38985b0a458e19eb4f4d89ccba